### PR TITLE
Add file headers to auto-generated Python sources.

### PR DESCRIPTION
**Pull request by Paebbels**
This PR modifies `pnodespy.py` to add an *auto-generated* note as a file header.

In addition, all 6 auto generated Python files where updated.

---------
This is ready for review.

### CODE REVIEW

**Comment by umarcor** [comment]
Ref #1569.

File: scripts/pnodespy.py [hunk 1]
```diff
@@ -37,40 +40,33 @@ def do_class_kinds():
...
         )
 
 
 def do_libghdl_elocations():
     classname = "vhdl__elocations"
-    print("from pyGHDL.libghdl import libghdl")
-    print()
+    print(dedent("""\
+        # Auto generated Python source file from Ada sources
+        # Call 'make' in 'src/vhdl' to regenerate:
```
Comment (umarcor) [inline scripts/pnodespy.py:60]: `xtools` does not exist now. Scripts are `scripts/pnodes.py` and `scripts/pnodespy.py`. I suggest rewording to: `Use scripts/pnodes*.py via make to regenerate:`

File: pyGHDL/libghdl/errorout.py [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# Auto generated Python source file from Ada sources
+# Use xtools via make to regenerate:
```
Comment (umarcor) [inline pyGHDL/libghdl/errorout.py:2]: `xtools` does not exist now. Scripts are `scripts/pnodes.py` and `scripts/pnodespy.py`. I suggest rewording to: `Use scripts/pnodes*.py via make to regenerate:`

Comment (Paebbels) [reply pyGHDL/libghdl/errorout.py:2]: After squashing my commits the headers didn't get update or other wired things happend in Git ...

That's what the script writes out, the header you referenced is an old version. I'll push an updated version.
```
# Auto generated Python source file from Ada sources
# Call 'make' in 'src/vhdl' to regenerate:
```

Comment (Paebbels) [reply pyGHDL/libghdl/errorout.py]: @umarcor I frorce-pushed (2ad051e). Please check again.

Comment (umarcor) [reply pyGHDL/libghdl/errorout.py]: LGTM!

**Comment by tgingold** [comment]
Thanks!
